PWD_Vault/endecrypt.py

```diff
@@ -1,5 +1,6 @@
 import os
 import base64
+import cryptography

```

Comment:
What's the need for this change? Was something broken?

Reply:
It produced an error in "decrypt" when i tried to implement the expect statement with (cryptography.fernet.InvalidToken)